api/api_public_blockchain.go

```diff
@@ -313,12 +296,26 @@ func DoCall(ctx context.Context, b Backend, args CallArgs, blockNrOrHash rpc.Blo
 	// this makes sure resources are cleaned up.
 	defer cancel()
 
-	// Add gas fee to sender for estimating gasLimit/computing cost or calling a function by insufficient balance sender.
-	state.AddBalance(msg.ValidatedSender(), new(big.Int).Mul(new(big.Int).SetUint64(msg.Gas()), msg.GasPrice()))
```

Comment:
Why did you remove this line?

Reply:
It was my misunderstood. I didn't fully understand the role of `EthStateOverride` in `api_ethereum.go`at that time.
I'll restore this line.